labs/lab4/chapter4.md

```diff
@@ -30,14 +30,14 @@ docker port wordpress
 
 and taking look at the "PORTS" column for the wordpress site. You 
 can also get your ip address by looking at the address for the 
-eth1 interface after you execute:
+eth0 interface after you execute:
 
 ```bash
-ip addr show dev eth1
+minishift ssh "ip addr show dev eth0"
```

Comment:
I thought the same thing, but I thought it might be confusing since that ip doesn't match the cdk.example.com ip. Either way is good with me.